packages/sst/support/signing-function/helpers/queryParameterBagToQueryString.ts

```diff
@@ -0,0 +1,32 @@
+import { QueryParameterBag } from "@aws-sdk/types";
+
+/**
+ * This function does not encodeURIComponent the values,
+ * because the result is meant to be used in a JS object.
+ */
+export const queryParameterBagToQueryString = (
+  query: QueryParameterBag
+): string => {
+  const keys: Array<string> = [];
+  const serialized: Record<string, string> = {};
+
+  for (const key of Object.keys(query)) {
```

Comment:
is `request.queryString` guaranteed to be an object, even if the url doesn't include it? Object.keys on null/undefined will throw an error.

Reply:
Yes, this is guaranteed by the  `queryStringToQueryParameterBag` function where [`query` is initialized with `const query = {}`](https://github.com/ealain/sst/blob/67128b3c02c1d5e65a3ea167551330186be7016b/packages/sst/support/signing-function/helpers/queryStringToQueryParameterBag.ts#L6).